### PR TITLE
RUN-14773 add configmap cli command

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/Researcher/cli-reference/runai-submit-dist-TF.md [hunk 1]
```diff
@@ -273,6 +273,10 @@ runai submit-dist tf --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Configure a `Configmap` object for use in a container.
```
Comment (shmuel-runai) [inline docs/Researcher/cli-reference/runai-submit-dist-TF.md:278]: > Configure a config 
sounds funny, doesn't it? you the expert
also not just a "use in a container" as there are other ways to use config map, suggest adding to the text something about it being a volume in the container

File: docs/Researcher/cli-reference/runai-submit-dist-mpi.md [hunk 2]
```diff
@@ -276,6 +276,10 @@ You can start an unattended mpi training Job of name dist1, based on Project *te
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Configure a `Configmap` object for use in a container.
```
Comment (shmuel-runai) [inline docs/Researcher/cli-reference/runai-submit-dist-mpi.md:281]: same

File: docs/Researcher/cli-reference/runai-submit-dist-pytorch.md [hunk 3]
```diff
@@ -280,6 +280,10 @@ runai submit-dist pytorch --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Configure a `Configmap` object for use in a container.
```
Comment (shmuel-runai) [inline docs/Researcher/cli-reference/runai-submit-dist-pytorch.md:285]: same

File: docs/Researcher/cli-reference/runai-submit-dist-xgboost.md [hunk 4]
```diff
@@ -268,6 +268,10 @@ runai submit-dist xgboost --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Configure a `Configmap` object for use in a container.
```
Comment (shmuel-runai) [inline docs/Researcher/cli-reference/runai-submit-dist-xgboost.md:273]: same

File: docs/Researcher/cli-reference/runai-submit.md [hunk 5]
```diff
@@ -346,6 +346,10 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 >
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Configure a `Configmap` object for use in a container.
```
Comment (shmuel-runai) [inline docs/Researcher/cli-reference/runai-submit.md:351]: same

**Review by shmuel-runai — COMMENTED** [review]
nice update
left a minor comment

File: docs/Researcher/cli-reference/runai-submit-dist-TF.md (hunk 1 above)
Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit-dist-TF.md:278]:
```suggestion
> Mlount a `Configmap` object for use as a data volume.
```

Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit-dist-TF.md:278]:
```suggestion
> Mlount a `Configmap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit-dist-mpi.md (hunk 2 above)
Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit-dist-mpi.md:281]:
```suggestion
> Mlount a `Configmap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit-dist-pytorch.md (hunk 3 above)
Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit-dist-pytorch.md:285]:
```suggestion
> Mlount a `Configmap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit-dist-xgboost.md (hunk 4 above)
Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit-dist-xgboost.md:273]:
```suggestion
> Mlount a `Configmap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit.md (hunk 5 above)
Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit.md:351]:
```suggestion
> Mlount a `Configmap` object for use as a data volume.
```

**Comment by jasonnovichRunAI** [comment]
@shmuel-runai added the suggested change. LMK if all ok please approve.

File: docs/Researcher/cli-reference/runai-submit-dist-TF.md [hunk 6]
```diff
@@ -273,6 +273,10 @@ runai submit-dist tf --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mlount a `Configmap` object for use as a data volume.
```
Comment (shmuel-runai) [inline docs/Researcher/cli-reference/runai-submit-dist-TF.md:278]: Mlount --> Mount 
here and everywhere

**Review by shmuel-runai — COMMENTED** [review]
minor type

File: docs/Researcher/cli-reference/runai-submit-dist-TF.md (hunk 6 above)
Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit-dist-TF.md:278]:
```suggestion
> Mount a `Configmap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit-dist-mpi.md [hunk 7]
```diff
@@ -276,6 +276,10 @@ You can start an unattended mpi training Job of name dist1, based on Project *te
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mlount a `Configmap` object for use as a data volume.
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/runai-submit-dist-mpi.md:281]:
```suggestion
> Mount a `Configmap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit-dist-pytorch.md [hunk 8]
```diff
@@ -280,6 +280,10 @@ runai submit-dist pytorch --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mlount a `Configmap` object for use as a data volume.
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/runai-submit-dist-pytorch.md:285]:
```suggestion
> Mount a `Configmap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit-dist-xgboost.md [hunk 9]
```diff
@@ -268,6 +268,10 @@ runai submit-dist xgboost --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mlount a `Configmap` object for use as a data volume.
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/runai-submit-dist-xgboost.md:273]:
```suggestion
> Mount a `Configmap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit.md [hunk 10]
```diff
@@ -346,6 +346,10 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 >
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mlount a `Configmap` object for use as a data volume.
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/runai-submit.md:351]:
```suggestion
> Mount a `Configmap` object for use as a data volume.
```

**Comment by jasonnovichRunAI** [comment]
@shmuel-runai fixed.

File: docs/Researcher/cli-reference/runai-submit-dist-TF.md [hunk 11]
```diff
@@ -273,6 +273,10 @@ runai submit-dist tf --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mount a `Configmap` object for use as a data volume.
```
Comment (shmuel-runai) [inline docs/Researcher/cli-reference/runai-submit-dist-TF.md:278]: Configmap --> ConfigMap
here and everywhere

**Review by shmuel-runai — APPROVED** [review]
Cool work
Approved, but please update to `ConfigMap`

File: docs/Researcher/cli-reference/runai-submit-dist-TF.md (hunk 11 above)
Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit-dist-TF.md:278]:
```suggestion
> Mount a `ConfigMap` object for use as a data volume.
```

Comment (jasonnovichRunAI) [reply docs/Researcher/cli-reference/runai-submit-dist-TF.md:278]:
```suggestion
> Mount a `ConfigMap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit-dist-mpi.md [hunk 12]
```diff
@@ -276,6 +276,10 @@ You can start an unattended mpi training Job of name dist1, based on Project *te
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mount a `Configmap` object for use as a data volume.
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/runai-submit-dist-mpi.md:281]:
```suggestion
> Mount a `ConfigMap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit-dist-pytorch.md [hunk 13]
```diff
@@ -280,6 +280,10 @@ runai submit-dist pytorch --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mount a `Configmap` object for use as a data volume.
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/runai-submit-dist-pytorch.md:285]:
```suggestion
> Mount a `ConfigMap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit-dist-xgboost.md [hunk 14]
```diff
@@ -268,6 +268,10 @@ runai submit-dist xgboost --name distributed-job --workers=2 -g 1 \
 > 
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mount a `Configmap` object for use as a data volume.
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/runai-submit-dist-xgboost.md:273]:
```suggestion
> Mount a `ConfigMap` object for use as a data volume.
```

File: docs/Researcher/cli-reference/runai-submit.md [hunk 15]
```diff
@@ -346,6 +346,10 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 >
 > Mount /root/data to NFS path /public/data on NFS server nfs.example.com for read-write access.
 
+#### --configmap-volume name=<name of configmap>,path=<path to mount> ...'
+
+> Mount a `Configmap` object for use as a data volume.
```
Comment (jasonnovichRunAI) [inline docs/Researcher/cli-reference/runai-submit.md:351]:
```suggestion
> Mount a `ConfigMap` object for use as a data volume.
```

**Pull request merged**